From f4e2838f664f4007838f4c635d7a4fe653aae68b Mon Sep 17 00:00:00 2001
From: Christoph Reiter <reiter.christoph@gmail.com>
Date: Mon, 9 Jan 2023 19:12:52 +0100
Subject: Fix build with clang

Building libnice with clang 15 fails with:

interfaces.c:913:12: error: incompatible pointer to integer conversion returning 'void *' from a function with result type 'guint' (aka 'unsigned int') [-Wint-conversion]
    return NULL;

Fix by returning 0 instead.
---
 agent/interfaces.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/agent/interfaces.c b/agent/interfaces.c
index 3590e0d..30944f5 100644
--- a/agent/interfaces.c
+++ b/agent/interfaces.c
@@ -910,7 +910,7 @@ nice_interfaces_get_if_index_by_addr (NiceAddress *addr)
 
   addresses = _nice_get_adapters_addresses ();
   if (!addresses)
-    return NULL;
+    return 0;
 
   for (a = addresses; a != NULL; a = a->Next) {
     /* Various conditions for ignoring the interface. */
-- 
cgit v1.2.1

